docs/measuring_performance/measuring_performance_rg1.md

```diff
@@ -0,0 +1,96 @@
+---
+---
+
+        <h3>
+          <a id="user-content-note-this-document-is-related-to-radargun-1x" class="anchor" href="#note-this-document-is-related-to-radargun-1x" aria-hidden="true"></a>Note: this document is related to RadarGun 1.x</h3>
```

Comment:
IMO, I don't think we should keep any docs about the 1.x release. I also had a **thought** about how to version the docs. I was thinking the menu could include links to previous versions of the docs that would be stored in a subdirectory of the `site` folder. For example, the 2.x docs could be accessed in the path `site/other_versions/2.x/index.html`. WDYT?